schema/deploy/roles/hcov19-visibility/create.sql

```diff
@@ -2,9 +2,6 @@
 
 begin;
 
-create role "hcov19-visibility";
-
-comment on role "hcov19-visibility" is
-    'Allows access to HCoV-19 presence/absence results';
+drop role "hcov19-visibility";
```

Comment:
Would this need to be `drop role if exists` so it could be rerun again even after the role has been removed, without throwing an error? (And similar question for the `view-owner` role)

Reply:
Yes, if it had to be rerun. But I'm not sure when that scenario would arise? The change would have to be `sqitch rework`ed, at which point I'd expect the `drop role` line to change anyway?

Reply:
Mhmm. Yes, that makes sense. Thanks!